cli/cmd/install-cni-plugin.go

```diff
@@ -178,12 +184,56 @@ func newCNIInstallOptionsWithDefaults() (*cniPluginOptions, error) {
 	return &cniOptions, nil
 }
 
-func (options *cniPluginOptions) buildValues() (*cnicharts.Values, error) {
+func buildValues(options *cniPluginOptions, valuesOverrides map[string]interface{}) (*cnicharts.Values, error) {
 	installValues, err := cnicharts.NewValues()
 	if err != nil {
 		return nil, err
 	}
 
+	if val, exists := valuesOverrides["enablePSP"]; exists {
+		installValues.EnablePSP = val.(bool)
+	}
+
+	if val, exists := valuesOverrides["podLabels"]; exists {
+		var podLabels map[string]string
+		if err := parseYAMLValue(val, &podLabels); err != nil {
+			return nil, err
+		}
+		installValues.PodLabels = podLabels
+	}
```

Comment:
This is more succint (also uses `ok`, which is more idiomatic in go):
```suggestion
	if val, ok := valuesOverrides["podLabels"]; ok {
		if err := parseYAMLValue(val, &installValues.PodLabels); err != nil {
			return nil, err
		}
	}

```